### ISSUE TITLE
System.Drawing.Imaging.Metafile(filename) throws wrong Exception when filename is Null on netcoreapp

### ISSUE THREAD

**Issue by KostaVlev** [dotnet/corefx#22598]
The test:

```c#
[ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
public void Ctor_NullString_ThrowsArgumentNullException()
{
    AssertExtensions.Throws<ArgumentNullException>("path", () => new Metafile((string)null));
}
```

**Expected:** `System.ArgumentNullException`

**Actual:** 
netfx `System.ArgumentNullException`
netcoreapp `System.ArgumentException`

(#20711)

**Comment by mellinoe** [comment]
I'm fixing this.

**Comment by KostaVlev** [comment]
Fixed by #22941.
